patchwork/models.py

```diff
@@ -823,6 +831,29 @@ class Meta:
         ]
 
 
+class Note(TimestampMixin, models.Model):
+    patch = models.ForeignKey(
+        Patch,
+        related_name='note',
+        related_query_name='note',
+        on_delete=models.CASCADE,
+    )
+    submitter = models.ForeignKey(User, on_delete=models.CASCADE)
+    content = models.TextField(null=False, blank=True)
+    maintainer_only = models.BooleanField(default=True)
```

Comment:
Are these not always maintainer-only? Why would we ever want to make them public?

Reply:
Not necessarily, it may be used to transmit additional context for reviewers that may not be adequate to have in the commit message, in this context it would make sense to have public notes.